core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/DataAccessSpec.scala

```diff
@@ -50,20 +53,41 @@ class DataAccessSpec extends TestDriverComponentWithFlatSpecAndMatchers with Sca
     withClue(
       "is the mysql against which these unit tests ran set up correctly with --character-set-server=utf8 or equivalent?"
     ) {
-      actual shouldBe "utf8"
+      actual shouldBe "utf8mb3"
+    }
+  }
+
+  it should "use utf8mb3_general_ci (via the utf8_general_ci alias) for MySQL's collation_server" in withEmptyTestDatabase {
+    /* See previous test for more details. utf8_general_ci is the collation associated with a character set of utf8.
+
+       Similar to how "utf8" is an alias for "utf8mb3", "utf8_general_ci" is an alias for "utf8mb3_general_ci". So, this
+       test looks for "utf8mb3_general_ci".
+     */
+    val collationLookup =
+      runAndWait(sql"""SHOW VARIABLES WHERE Variable_name = 'collation_server';""".as[(String, String)])
+    collationLookup should have size 1
+    val actual = collationLookup.head._2
+    info(s"actual collation set: $actual")
+    withClue(
+      "is the mysql against which these unit tests ran set up correctly with --collation_server=utf8_general_ci or equivalent?"
+    ) {
+      actual shouldBe "utf8mb3_general_ci"
     }
   }
 
-  it should "be testing against MySQL 5.7" in withEmptyTestDatabase {
+  // The docker version installed on DSP Jenkins does not like the "mysql" docker image, so we use "mysql/mysql-server"
+  //    instead, which only has a 8.0.32 tag. See /docker/run-mysql.sh.
+  //    TODO: Move to mysql:8.0.35 once we no longer depend on Jenkins.
```

Comment:
Is this noted in the move-off-jenkins ticket?

Reply:
yes, it is! … now :)